mass_mailing_switzerland/models/mass_mailing.py

```diff
@@ -163,7 +182,15 @@ class Mail(models.Model):
 
     @job(default_channel='root.mass_mailing')
     @related_action(action='related_action_emails')
-    @api.multi
+    @api.one
```

Comment:
`api.one` decorators are deprecated. Restore the multi tag and add a for loop to handle all cases:
```
for email in self:
    if email.mailing_id.mailing_slug:
...
```